vyper/ast/__init__.py

```diff
@@ -0,0 +1,66 @@
+from vyper.ast.nodes import (  # noqa: F401
```

Comment:
Is there any reason we can't just batch import, and move any extraneous functionality we don't want as part of the API for this stage to another module?

Reply:
I worry that a wildcard import opens the door to unexpected namespace issues.  If we use `from vyper.ast.nodes import *` we also import things like `itertools.chain` and `vyper.utils.annotate_source_code`.

Reply:
There's gotta be a fancy way to do this important to just grab all subclasses of `VyperNode`

Reply:
Yes, if you'd prefer I can definitely do some voodoo.

Reply:
ben casts `sys.modules[__name__]` + `cls.__mro__`...  it's super effective!